test_flight_paths.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+def test_extract_data_from_json():
+    """Test extracted data."""
+    from flight_paths import extract_data_from_json
+    data = extract_data_from_json('cities_with_airports.json')
+    assert len(data) > 0
+    assert type(data) is list
+    assert type(data[0]) is dict
+
+
+def test_build_graph():
```

Comment:
This is not a test you should really need. Your graph is coming in more or less unmodified from your data structures, no?